evaluation_experiments/EXP3_cibersort_eval.ipynb

```diff
@@ -0,0 +1,6160 @@
+{
```

Comment:
<p>Either in the begining or here, or somewhere else you think it's appropriate, could you include a description of the difference between scaled and non-scaled and why you use both?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/pred_missing_celltypes/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>